src/cascadia/TerminalApp/AppKeyBindings.cpp

```diff
@@ -1,117 +1,294 @@
+static const std::wstring PASTETEXT_KEY{ L"paste" };
+static const std::wstring NEWTAB_KEY{ L"newTab" };
+static const std::wstring NEWTABWITHPROFILE0_KEY{ L"newTabProfile0" };
+static const std::wstring NEWTABWITHPROFILE1_KEY{ L"newTabProfile1" };
+static const std::wstring NEWTABWITHPROFILE2_KEY{ L"newTabProfile2" };
+static const std::wstring NEWTABWITHPROFILE3_KEY{ L"newTabProfile3" };
+static const std::wstring NEWTABWITHPROFILE4_KEY{ L"newTabProfile4" };
+static const std::wstring NEWTABWITHPROFILE5_KEY{ L"newTabProfile5" };
+static const std::wstring NEWTABWITHPROFILE6_KEY{ L"newTabProfile6" };
+static const std::wstring NEWTABWITHPROFILE7_KEY{ L"newTabProfile7" };
+static const std::wstring NEWTABWITHPROFILE8_KEY{ L"newTabProfile8" };
+static const std::wstring NEWTABWITHPROFILE9_KEY{ L"newTabProfile9" };
+static const std::wstring NEWWINDOW_KEY{ L"newWindow" };
+static const std::wstring CLOSEWINDOW_KEY{ L"closeWindow" };
+static const std::wstring CLOSETAB_KEY{ L"closeTab" };
+static const std::wstring SWITCHTOTAB_KEY{ L"switchToTab" };
+static const std::wstring NEXTTAB_KEY{ L"nextTab" };
+static const std::wstring PREVTAB_KEY{ L"prevTab" };
+static const std::wstring INCREASEFONTSIZE_KEY{ L"increaseFontSize" };
+static const std::wstring DECREASEFONTSIZE_KEY{ L"decreaseFontSize" };
+static const std::wstring SCROLLUP_KEY{ L"scrollUp" };
+static const std::wstring SCROLLDOWN_KEY{ L"scrollDown" };
+
+static const std::vector<std::pair<ShortcutAction, std::wstring>> commandNames {
+    { ShortcutAction::CopyText, COPYTEXT_KEY },
+    { ShortcutAction::PasteText, PASTETEXT_KEY },
+    { ShortcutAction::NewTab, NEWTAB_KEY },
+    { ShortcutAction::NewTabProfile0, NEWTABWITHPROFILE0_KEY },
+    { ShortcutAction::NewTabProfile1, NEWTABWITHPROFILE1_KEY },
+    { ShortcutAction::NewTabProfile2, NEWTABWITHPROFILE2_KEY },
+    { ShortcutAction::NewTabProfile3, NEWTABWITHPROFILE3_KEY },
+    { ShortcutAction::NewTabProfile4, NEWTABWITHPROFILE4_KEY },
+    { ShortcutAction::NewTabProfile5, NEWTABWITHPROFILE5_KEY },
+    { ShortcutAction::NewTabProfile6, NEWTABWITHPROFILE6_KEY },
+    { ShortcutAction::NewTabProfile7, NEWTABWITHPROFILE7_KEY },
+    { ShortcutAction::NewTabProfile8, NEWTABWITHPROFILE8_KEY },
+    { ShortcutAction::NewTabProfile9, NEWTABWITHPROFILE9_KEY },
+    { ShortcutAction::NewWindow, NEWWINDOW_KEY },
+    { ShortcutAction::CloseWindow, CLOSEWINDOW_KEY },
+    { ShortcutAction::CloseTab, CLOSETAB_KEY },
+    { ShortcutAction::SwitchToTab, SWITCHTOTAB_KEY },
+    { ShortcutAction::NextTab, NEXTTAB_KEY },
+    { ShortcutAction::PrevTab, PREVTAB_KEY },
+    { ShortcutAction::IncreaseFontSize, INCREASEFONTSIZE_KEY },
+    { ShortcutAction::DecreaseFontSize, DECREASEFONTSIZE_KEY },
+    { ShortcutAction::ScrollUp, SCROLLUP_KEY },
+    { ShortcutAction::ScrollDown, SCROLLDOWN_KEY },
+};
+
+namespace winrt::TerminalApp::implementation
+{
+    void AppKeyBindings::SetKeyBinding(TerminalApp::ShortcutAction const& action,

```

Comment:
Our coding guidelines expect `const Blah&` not `Blah const&`

Reply:
I blame cppwinrt's auto code generation :P

Reply:
Out of curiosity: Why is it Blah const& in some other files, like TermControl.cpp? Is this just old code or is there another reason (like another team worked on the TerminalControl project)?

Reply:
This is usually a side-effect of the auto code generation of the cppwinrt toolset. When you define methods in the `.idl` of a class, you can run a build, and cppwinrt will generate a stub version of your class's `.cpp` and `.h`, with all your methods, properties, etc properly defined, in the `Generated Files/` directory. I'll usually just grab the definitions from those files and paste them into the real implementation instead of trying to write them myself.

The side effect however, is that they generate `Blah const&` instead of `const Blah&`, and we don't always catch that in code review. And as soon as one makes it through PR, it's likely to get copied and pasted into another method definition somewhere else in the file.

Reply:
Makes sense. Thank you for the explanation.